opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -118,144 +152,121 @@ message MetricDescriptor {
-  // all data points.
-  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 5;
-}
-
-// Int64DataPoint is a single data point in a timeseries that describes the time-varying
-// values of a int64 metric.
-message Int64DataPoint {
-  // The set of labels that uniquely identify this timeseries.
-  repeated opentelemetry.proto.common.v1.StringKeyValue labels = 1;
-
-  // start_time_unix_nano is the time when the cumulative value was reset to zero.
-  // This is used for Counter type only. For Gauge the value is not specified and
-  // defaults to 0.
-  //
-  // The cumulative value is over the time interval (start_time_unix_nano, time_unix_nano].
-  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
-  //
-  // Value of 0 indicates that the timestamp is unspecified. In that case the timestamp
-  // may be decided by the backend.
-  fixed64 start_time_unix_nano = 2;
-
-  // time_unix_nano is the moment when this value was recorded.
-  // Value is UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
-  fixed64 time_unix_nano = 3;
+  // Temporality describes how metric values are reported over time.
+  enum Temporality {
+    // UNSPECIFIED is the default Temporality, it MUST not be used.
+    UNSPECIFIED = 0;
 
-  // value itself.
-  int64 value = 4;
-}
+    // INSTANTANEOUS is a metric whose values are measured at a particular
+    // instant.
+    //
+    // If an INSTANTANEOUS metric is reported over any time interval the
+    // values are representative of distinct measurements within the
+    // interval.
+    INSTANTANEOUS = 1;
+
+    // DELTA is a metric whose values are measured and successively
+    // aggregated over a reporting interval. The reported value represents
+    // the change from the previous interval.
+    //
+    // Each reported value for a DELTA metric over a time interval is
+    // self-contained, the values do not depend on values in other time
+    // intervals. This is unlike a CUMMULATIVE metric where values are sums
+    // of all measured events in the time interval and all previously
+    // measured values of the metric.
+    //
+    // DELTA metrics are not guaranteed to be monotonic.
+    DELTA = 2;
+
+    // CUMULATIVE is a metric whose values are measured and successively
+    // added to the lifetime total of the metric.
+    //
+    // Each reported value for a CUMULATIVE metric is the sum of all
+    // measurements up to and including that one.
+    //
+    // CUMULATIVE metrics are not guaranteed to be monotonic.
+    CUMULATIVE = 3;
+  }
+  Temporality temporality = 5;
+
+  // Monotonic describes the metric values to be entirely non-increasing,
+  // or entirely non-decreasing.
+  enum Monotonic {
+    // UNKNOWN is the default, and means it the monotonic nature of the
+    // metric values is known.
+    UNKNOWN = 0;
+
+    // NONINCREASING means all the successive metric values decrease or
+    // remain constant.
+    NONINCREASING = 1;
```

Comment:
Do we need to support this mode? It seems that we can simply ask users to use the "-value" when recording and we have to deal with only one type.

Reply:
removed.